Tests/scripts/hook_validations/description.py

```diff
@@ -22,16 +22,54 @@ def is_valid(self):
 
         return self._is_valid
 
+    def is_valid_beta_description(self):
+        """Check if beta disclaimer exists in detailed description"""
+        data_dictionary = get_yaml(self.file_path)
+        description_in_yml = data_dictionary.get('detaileddescription', '') if data_dictionary else ''
+
+        if not re.match(BETA_INTEGRATION_REGEX, self.file_path, re.IGNORECASE):
+            package_path = os.path.dirname(self.file_path)
+            try:
+                md_file_path = glob.glob(os.path.join(os.path.dirname(self.file_path), '*_description.md'))[0]
+            except IndexError:
+                self._is_valid = False
+                print_error("No detailed description file was found in the package {}. Please add one,"
+                            " and make sure it includes the beta disclaimer note."
+                            "It should contain the string in constant"
+                            "\"BETA_INTEGRATION_DISCLAIMER\"".format(package_path))
+                return False
+
+            with open(md_file_path) as description_file:
+                description = description_file.read()
+            if BETA_INTEGRATION_DISCLAIMER not in description:
```

Comment:
The code is similar but not the same. one is for the package and the other is for the detailed description field in an an packed integration. 
The error message is also different because of that.